charts/redpanda/templates/servicemonitor.yaml

```diff
@@ -38,10 +38,22 @@ spec:
     scheme: https
     tlsConfig:
       insecureSkipVerify: true
+      ca:
+        secret:
+          name: {{ template "redpanda.fullname" $ }}-default-cert
```

Comment:
This is the default name in most cases, but not necessarily, we should check to see if a secret is being provided instead also.